csrankings-x.csv

```diff
@@ -336,3 +336,4 @@ Xun Wang,University of Technology Sydney,https://www.uts.edu.au/staff/xun.wang-1
 Xun Yi,RMIT University,https://www.rmit.edu.au/contact/staff-contacts/academic-staff/y/yi-professor-xun,oDRazncAAAAJ
 Xusheng Xiao,Case Western Reserve University,http://engineering.case.edu/groups/xusheng-xiao,qVx6V7UAAAAJ
 Xuyun Zhang,Macquarie University,https://researchers.mq.edu.au/en/persons/xuyun-zhang,wbF6HL8AAAAJ
+Xucong Zhang, TU Delft, https://www.ccmitss.com/zhang

```

Comment:
Remove spaces after commas, add Google Scholar ID.

Reply:
Thank you for checking! Sorry that I did not write it correctly. Please check the changes.